src/main/java/org/dave/compactmachines3/tile/TileEntityFieldProjector.java

```diff
@@ -368,7 +363,7 @@ public void update() {
             // Remove blocks from the world
             for(BlockPos pos : getInsideBlocks()) {
                 world.setBlockToAir(pos);
-                ((WorldServer)world).spawnParticle(EnumParticleTypes.SMOKE_LARGE, pos.getX(), pos.getY(), pos.getZ(), 10, 0.5D, 0.5D, 0.5D, 0.01D, new int[0]);
+                ((WorldServer)world).spawnParticle(EnumParticleTypes.SMOKE_LARGE, pos.getX(), pos.getY(), pos.getZ(), 10, 0.5D, 0.5D, 0.5D, 0.01D);
```

Comment:
Why the removal of the int array here?

Reply:
The method was obviously generated by autocomplete. It's a varargs array and omitting it is more idiomatic.